languages/en-US/administration/configuration.json

```
@@ -2,34 +2,34 @@
     "DESCRIPTION": "Shows the server configuration!",
     "USAGE": "{{prefix}}configuration",
     "EXAMPLES": "{{prefix}}configuration",
-    "PREFIX_TITLE": "Server prefix",
-    "IGNORED_CHANNELS_TITLE": "Ignored channel(s)",
+    "PREFIX_TITLE": "\n- Server prefix:",
+    "IGNORED_CHANNELS_TITLE": "- Ignored channel(s):",
     "NO_IGNORED_CHANNELS": "No ignored channels.",
-    "AUTOROLE_TITLE": "Autorole",
+    "AUTOROLE_TITLE": "- Autorole:",
     "AUTOROLE_CONTENT": "Role: {{roleName}}",
     "AUTOROLE_DISABLED": "Autorole disabled.",
-    "WELCOME_TITLE": "Welcome",
+    "WELCOME_TITLE": "- Welcome:",
     "WELCOME_CONTENT": "Channel: {{channel}}\nImage: {{withImage}}",
     "WELCOME_DISABLED": "Welcome messages disabled.",
-    "GOODBYE_TITLE": "Goodbye",
+    "GOODBYE_TITLE": "- Goodbye:",
     "GOODBYE_CONTENT": "Channel: {{channel}}\nImage: {{withImage}}",
     "GOODBYE_DISABLED": "Goodbye messages disabled.",
     "SPECIAL_CHANNELS": "Special channels",
-    "MODLOGS": "Moderation logs: {{channel}}",
-    "FORTNITESHOP": "Fortnite daily shop: {{channel}}",
-    "SUGGESTIONS": "Suggestions: {{channel}}",
-    "REPORTS": "Reports: {{channel}}",
-    "AUTOMOD_TITLE": "Auto-moderation",
-    "AUTOMOD_CONTENT": "Auto-moderation enabled. Ignored channels: {{channels}}",
+    "MODLOGS": "Moderation logs: *{{channel}}*",
+    "FORTNITESHOP": "Fortnite daily shop: *{{channel}}*",
+    "SUGGESTIONS": "Suggestions: *{{channel}}*",
+    "REPORTS": "Reports: *{{channel}}*",
+    "AUTOMOD_TITLE": "- Auto-moderation:",
```

Sans le tiret c'est plus propre visuellement 
```suggestion
    "AUTOMOD_TITLE": "Auto-moderation:",
```